xchange-upbit/pom.xml

```diff
@@ -28,7 +28,10 @@
             <artifactId>xchange-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.auth0</groupId>
+            <artifactId>java-jwt</artifactId>
```

Comment:
can you define this in the parent pom instead?

Reply:
yes, I changed location. and java-jwt version upgraded.